jobs/env/ci-cri-containerd-e2e-gce.env

```diff
@@ -10,8 +10,8 @@ TEST_ETCD_VERSION=2.2.1
 ENABLE_POD_SECURITY_POLICY=true
 
 # Envs for cri-containerd.
-KUBE_MASTER_EXTRA_METADATA=user-data=/go/src/github.com/kubernetes-incubator/cri-containerd/test/e2e/master.yaml,cri-containerd-configure-sh=/go/src/github.com/kubernetes-incubator/cri-containerd/test/configure.sh
-KUBE_NODE_EXTRA_METADATA=user-data=/go/src/github.com/kubernetes-incubator/cri-containerd/test/e2e/node.yaml,cri-containerd-configure-sh=/go/src/github.com/kubernetes-incubator/cri-containerd/test/configure.sh
+KUBE_MASTER_EXTRA_METADATA=user-data=github.com/kubernetes-incubator/cri-containerd/test/e2e/master.yaml,cri-containerd-configure-sh=github.com/kubernetes-incubator/cri-containerd/test/configure.sh
```

Comment:
then the relative path will be `../github.com/...`?

Reply:
I'm completely confused. :p 
So I think the safest way is to specify `--root=/go/src`, and use abusolute path here?

Reply:
@krzyzacy Changed to absolute path `/workspace/github.com/XXX`.

This should work now.